src/views/homepage/index.tsx

```diff
@@ -64,6 +64,5 @@ function HomePageView(): ReactElement {
  */
 HomePageView.contentType = 'tutorials'
 HomePageView.layout = BaseNewLayout
-HomePageView.theme = GlobalThemeOption.light
```

Comment:
@kendallstrautman was this the last remaining instance where we set `.theme` on a page component?